sync-github-vsts.yml

```diff
@@ -0,0 +1,44 @@
+# variables:
+  # branches: defined on azure portal, separated by '|'
+  # vstsUrl: defined on azure portal
+
+trigger:
+  batch: true
+  branches:
+    include:
```

Comment:
Now that we have many pipelines (and potentially more after OneBranch), consider move all azure pipeline yaml to the `.pipelines` folder to match OneBranch convention.

Reply:
Prefer to do it when invloving OneBranch

Reply:
Consider change the file name to `sync-github-to-azure-repos.yml`